lib/symengine.rb

```diff
@@ -33,6 +33,29 @@ def Function(n)
     def evalf(operand, prec=53, real=false)
         return _evalf(operand, prec, real)
     end
+    def evalf_dirichlet_eta(exp)
+        SymEngine::evalf(SymEngine::dirichlet_eta(exp))
+    end
+    def evalf_zeta(exp)
+        SymEngine::evalf(SymEngine::zeta(exp))
+    end
+    def lambdify_code(exp)
+      sym_map = exp.free_symbols.map { |sym| sym.to_s}.sort.join(",")
```

Comment:
Getting a symbol array is a great idea.
Problem with free_symbols is that the order is architecture dependent.  For example, in `cylinder_volume = SymEngine::PI * SymEngine::Symbol.new(:r) ** 2 * SymEngine::Symbol.new(:h)`, free_symbols is a set whose order is unspecified in SymEngine and it could be either `r, h` or `h, r` depending on your os and toolchain.

So, auto generating the order is going to hurt reproducibility and reproducibility is something we should be careful about.